doc/azure2az_commands.rst

```diff
@@ -0,0 +1,305 @@
+azure storage account sas create                             az storage account generate-sas
+azure storage account set                                    az storage account update
+azure storage account show                                   az storage account show
+azure storage account usage show                             az storage account show-usage
+azure storage blob copy start                                az storage blob copy start
+azure storage blob copy stop                                 az storage blob copy cancel
+azure storage blob delete                                    az storage blob delete
+azure storage blob download                                  az storage blob download
+azure storage blob lease acquire                             az storage blob lease acquire
+azure storage blob lease break                               az storage blob lease break
+azure storage blob lease change                              az storage blob lease change
+azure storage blob lease release                             az storage blob lease release
+azure storage blob lease renew                               az storage blob lease renew
+azure storage blob list                                      az storage blob list
+azure storage blob sas create                                az storage blob generate-sas
+azure storage blob show                                      az storage blob show
+azure storage blob snapshot                                  az storage blob snapshot
+azure storage blob update                                    az storage blob update
+azure storage blob upload                                    az storage blob upload
+azure storage container create                               az storage container create
+azure storage container delete                               az storage container delete
+azure storage container lease acquire                        az storage container lease acquire
+azure storage container lease break                          az storage container lease break
+azure storage container lease change                         az storage container lease change
+azure storage container lease release                        az storage container lease release
+azure storage container lease renew                          az storage container lease renew
+azure storage container list                                 az storage container list
+azure storage container policy create                        az storage container policy create
+azure storage container policy delete                        az storage container policy delete
+azure storage container policy list                          az storage container policy list
+azure storage container policy set                           az storage container policy update
+azure storage container policy show                          az storage container policy show
```

Comment:
Similar to how you did for the DNS records, you could shorten this with:

`azure storage [container|table|queue] policy    az storage [container|table|queue] policy`